src/main/java/io/pravega/connectors/flink/ReaderCheckpointHook.java

```diff
@@ -98,13 +98,13 @@ public String getIdentifier() {
         return checkpointResult;
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     public void restoreCheckpoint(long checkpointId, Checkpoint checkpoint) throws Exception {
         // checkpoint can be null when restoring from a savepoint that
         // did not include any state for that particular reader name
         if (checkpoint != null) {
-            this.readerGroup.resetReadersToCheckpoint(checkpoint);
+            this.readerGroup.resetReaderGroup(ReaderGroupConfig.builder().disableAutomaticCheckpoints().
```

Comment:
The changes of the API should be first merged in `master` and then cherry picked to 0.6.2 if necessary. The removal of deprecated APIs in Pravega will come in 0.7.